File: Selnium script for testing/Script with Python/test_scenario_18.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_18():
    """
    Dummy test run for scenario 18
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_18")

    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.ID, "user_name").send_keys("test_data_107")
    driver.find_element(By.ID, "user_name").send_keys("test_data_193")
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_861")
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_355")
    driver.find_element(By.NAME, "password_field").send_keys("test_data_359")
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_857")
    driver.find_element(By.NAME, "password_field").send_keys("test_data_603")
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    time.sleep(2)

    driver.quit()

if __name__ == "__main__":
    execute_test_18()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario18.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario18 {
    public static void executeTest18() {
        /*
         * Dummy test run for scenario 18
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_18");

        try {
            Thread.sleep(2000);
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_599");
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.xpath("//button[@type='submit']")).click();
            driver.findElement(By.linkText("Log Out")).click();
            Thread.sleep(2000);
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_402");
            driver.findElement(By.linkText("Log Out")).click();
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            Thread.sleep(2000);
            driver.findElement(By.linkText("Log Out")).click();
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.linkText("Log Out")).click();
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest18();
    }
}
